### DISCUSSION TITLE
Make user fields translatable

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9083] [Ideas]
> Created by: Patrick Nesbitt (patrick@43north.jp) on 2015/03/13 00:43:03 +0000
> Votes at time of UserVoice import: 4

-----------------------------------

It would be great if fields allocated to users were translatable. At the moment, custom fields need to be set up for each language.

**Comment by brandonkelly** [comment]
Related SE question: http://craftcms.stackexchange.com/questions/510/fields-applied-to-the-user-area-are-not-translatable/511

**Comment by medoingthings** [comment]
+1 Would be massively helpful. Otherwise I had to duplicate a huge amount of custom fields for each language or move them into entries, which would be unpleasant.

**Comment by mijar** [comment]
Would love this too!

**Comment by stevenvandemoortele** [comment]
+1 Ran in to this issue too.

**Comment by mateostabio** [comment]
This is my road block right now... Here in Quebec everything has to be bilingual EN & FR, including users department, users job title and users bios. Any idea of when this will be added?? or do I have to copy/paste everything into a structure  -_-  Actually no, because then any user can edit anyone's bio within the structure... darn!

**Comment by tommysvr** [comment]
Adding to this request, I’m building a Japanese/English site and require biographies and names in both languages. Currently getting around this by using a category, but it’s not great to have the same/very similar content twice.
